token-stakedrop/README.adoc

```diff
@@ -1,70 +1,87 @@
-# KEEP Token Tracker
+# Overview
 
-This is a tool for tracking KEEP Token ownership. It will walk over predefined
-sources of truth and inspect KEEP Token ownership at a specific block number.
+This is a tool for finding KEEP token owners on Ethereum (or on any other 
+EVM-compatible chain). It will walk over predefined sources of truth
+(information) and lookup KEEP token owners at a specific block number.
+The tool generates a Merkle tree used as an input to
+https://github.com/keep-network/keep-core/blob/master/solidity/contracts/token-distribution/TokenDistributor.sol[`TokenDistributor`]
+contract airdropping another ERC20 token with amounts based on KEEP token 
```

Comment:
... with amounts equal to KEEP token balances.

Reply:
Not sure if "equals" work here - some sources, like LPs staked are more complicated. WDYT?

Reply:
Okay, I see. I think when we say "based" in this context is generic and whoever reads it, doesn't know how much they will get, might be 1 but maybe other number. When we say "equal", then it's 1:1. In the overview we just say that whatever KEEP we found (doesn't matter how) then we will sort match it.
In the LP Rewards description we go into the more details about how we find KEEP tokens, and the finding is "based" on the LP tokens.
Idk.. if you think it's okay to use "based" I'm okay with it too :)

Reply:
Yeah, I'd keep "based" for the exact reason you mentioned, I'd keep the precise information under the particular source of truth and be more generic earlier.